airbyte-integrations/connectors/source-zendesk-support/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,1928 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "group_memberships",
+        "json_schema": {
```

Comment:
You can leave the json schema empty in this file. It will simplify working with it. (It was a recommendation from Eugene Kulak)

Reply:
Done